Mini_DBMS/Mini_DBMS/Controllers/HomeController.cs

```diff
@@ -42,16 +42,13 @@ public ActionResult CreateDatabase(Database database)
 
         public ActionResult CreateTable(Table table)
         {
+            foreach (var t in currentDatabase.Tables)
```

Comment:
If you really want to make validation, make it visible on UI also, not just a refresh and nothing happen...